configure.ac

```diff
@@ -397,7 +397,7 @@ AC_ARG_ENABLE(request-early,
 
 # Enable duplicate rules id
 AC_ARG_ENABLE(unique_id,
-              AS_HELP_STRING([--enable-unique_id],
+              AS_HELP_STRING([--enable-not_unique_id],
```

Comment:
Hum, I think it will be best if we can have here something like:

--enable-rule-id-validation 

The help will be: Enable checking for the existence of an unique id (default option)

Usually we list the --enable- help, and we assume that --disable will be the opposite.

What do you think?